client/routers/CalendarRouter/Calendar/container.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import fetchTodos from './../../../actions/fetchTodos'
+import chooseTodo from './../../../actions/chooseTodo';
+
+function mapStateToProps(state){
+  return {
+    todos: state.todos,
+    chosenTodoId: state.chosenTodoId,
+    categories: state.categories,
+    chosenCategoryId: state.chosenCategoryId,
+    colorMap: state.colorMap
```

Comment:
do we use colorMap? If not I'd suggest to remove it from here.

Reply:
i was trying some things. It will be usefull later